docker-compose.demo.uffizzi.yml

```diff
@@ -0,0 +1,41 @@
+# This compose file is for demonstration only, do not use in prod.
+version: "3.9"
+
+x-uffizzi:
+  ingress:
+    service: app
+    port: 3001
+  continuous_previews:
+    deploy_preview_when_pull_request_is_opened: true
+    delete_preview_when_pull_request_is_closed: true
+    share_to_github: true
```

Comment:
What does this config do?

Reply:
This is not used. Will remove. (It's for pull request commenting)